definitions/json/CaseEvent/CaseEvent.json

```
@@ -584,8 +584,8 @@
   {
     "CaseTypeID": "ET_Scotland",
     "ID": "pseRespondentViewNotification",
-    "Name": "Judgement, Order, Notification",
-    "Description": "View a judgment, order or notification",
+    "Name": "Judgment, Order, Notification",
+    "Description": "View a judgment, order and notification",
```

I missed this on my suggested change, should've been or instead of and
```suggestion
    "Description": "View a judgment, order or notification",
```